### PR TITLE
OCPBUGS-32400: base OIDC status updates on the ExternalOIDC feature gate

### PR DESCRIPTION
**Pull request by stlaz**
/assign @deads2k 
/cc @jhadvig

### CODE REVIEW

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -504,6 +517,16 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		informer.Start(ctx.Done())
 	}
 
+	go featureGateAccessor.Run(ctx)
```
Comment (deads2k) [inline pkg/console/starter/starter.go:520]: move this right under the creation on new line 191

File: pkg/console/starter/starter.go [hunk 2]
```diff
@@ -245,13 +257,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
...
 		configInformers.Config().V1().Authentications(),
+		configInformers.Config().V1().FeatureGates(),
 		configClient.ConfigV1().Authentications(),
 		operatorConfigInformers.Operator().V1().Consoles(),
-		apiextensionsInformers.Apiextensions().V1().CustomResourceDefinitions(),
 		kubeInformersConfigNamespaced.Core().V1().ConfigMaps(),
 		kubeInformersNamespaced.Core().V1().Secrets(),
 		kubeInformersNamespaced.Core().V1().ConfigMaps(),
 		kubeInformersNamespaced.Apps().V1().Deployments(),
+		featureGateAccessor,
```
Comment (deads2k) [inline pkg/console/starter/starter.go:267]: pass a gate value here.  like this: https://github.com/openshift/enhancements/blob/06fce4b637487078bc9834378cb8d4a4cb055ae6/dev-guide/featuresets.md?plain=1#L190-L195

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**